.github/workflows/e2e.ci.yaml

```diff
@@ -27,9 +27,12 @@ on:
 env:
   SKIP_TEST: true
   ISTIO_VERSION: 1.7.1
-  TAG: ${{ github.sha }}
-  SCRIPTS_DIR: main/test/e2e-mesh/e2e-istio/scripts
-  SW_OAP_BASE_IMAGE: openjdk:8-jre-alpine
+  OAP_TAG: cef4f6dcac5b14cd67f51a64ad08052bbc046653
+  OAP_REPO: ghcr.io/apache/skywalking/oap
+  UI_TAG: cef4f6dcac5b14cd67f51a64ad08052bbc046653
+  UI_REPO: ghcr.io/apache/skywalking/ui
+  SATELLITE_TAG: v64f1938e118e95b8d8f50756f666e4777ce6aca2
+  SATELLITE_REPO: ghcr.io/apache/skywalking-satellite/skywalking-satellite
```

Comment:
Move these into the e2e environment file

Reply:
Do you mean only move the satellite to E2E environment config? Because the oap and ui, these image have different tag and repo.

Reply:
I think the point is where we define the tags. If repositories are expected to reused in other e2e, then it is better to move together.

Reply:
ok, I think we could move these tags and repo to a single e2e file. using a matrix to run them.

Reply:
If we need tests through matrix, that works.

Reply:
How is this resolved ? The env vars are still here